source/includes/fundamentals/code-snippets/libmongocrypt-gradle-versioned.rst

```diff
@@ -0,0 +1,6 @@
+.. code-block:: groovy
+
+   dependencies {
+       compile 'org.mongodb:mongodb-crypt:1.1.0-beta1'
```

Comment:
If you're not going to publish this for a while, you can just change to to 1.1.0